server.dev.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
```

Comment:
Its for local development, it sets up webpack-dev-middleware for hot reloading on the client and mounts the server app for server-side hot loading too.